mongo/options/dropcollectionoptions.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package options
+
+// DropCollectionOptions represents options that can be used to configure a Drop operation.
+type DropCollectionOptions struct {
+	// EncryptedFields configures encrypted fields for encrypted collections.
+	//
+	// This option is only valid for MongoDB versions >= 6.0
+	EncryptedFields interface{}
```

Comment:
Is there any more meaningful type representation for `EncryptedFields`? `interface{}` can certainly be helpful if there is a need for forward compatibility. But, what exactly is the structure of `encryptedFields` on the server? `map[string]string`?